src/common/stream.cc

```diff
@@ -30,6 +30,9 @@ FileBuffer::FileBuffer(const std::string& fileName)
 void
 FileBuffer::init(const char* fileName)
 {
+    if (!fileName) {
+        throw std::runtime_error("Wrong file name");
```

Comment:
Suggest change this text to "Missing fileName".

Reply:
@dennisjenkins75 I've made this change